packages/platform/atoms/hooks/usePublicEvent.tsx

```diff
@@ -30,6 +37,12 @@ export const usePublicEvent = (props: { username: string; eventSlug: string }) =
         })
         .then((res) => {
           if (res.data.status === SUCCESS_STATUS) {
+            if (props.duration && res.data.data) {
+              // note(Lauris): In case of "dynamic" event type default event duration returned by the API is 30,
```

Comment:
This caught my eye - dynamic event also changes a few things like the location (defaults to the first username in the lists "default conferencing" type)

Not sure if we need to take that into account here also

Reply:
Good note, but for now it looks like it should be fine as is.